devops/openshift/api-deploy.yml

```diff
@@ -168,6 +168,16 @@ objects:
                     secretKeyRef:
                       key: ${ZONE_B_SFTP_PRIVATE_KEY_NAME_KEY}
                       name: ${ZONE_B_SFTP_SECRET_NAME}
+                - name: MSFAA_REQUEST_FOLDER
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${MSFAA_REQUEST_FOLDER_NAME_KEY}
+                      name: ${API_SECRET_NAME}
```

Comment:
Just to know why is name of key : ${MSFAA_REQUEST_FOLDER_NAME_KEY} is API_SECRET_NAME

Correct me if I am missing something